src/attributes/attr.js

```diff
@@ -3,11 +3,14 @@ define([
 	"../core/access",
 	"./support",
 	"../var/rnotwhite",
+	"./val",
```

Comment:
My guess is it's leftover from a prop hook for retrieving button value or something. It doesn't seem necessary anymore.